.github/workflows/build_gradle.yml

```
@@ -63,4 +63,111 @@ jobs:
+          MARIADB_PASSWORD: test
+          MARIADB_DATABASE: test
+          MARIADB_ROOT_PASSWORD: test
+          POSTGRES_USER: test
+          POSTGRES_PASSWORD: test
+          POSTGRES_DB: test
+        options: >-
+          --health-cmd "${{ matrix.wait }}"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        command: ${{ matrix.cmd }}
```

actionlint (1.7.12)

[error] unexpected key "command" for "services" section. expected one of "credentials", "env", "image", "options", "ports", "volumes"

(syntax-check)